examples/sensor-data-java/templates/nfs.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pv-volume
```

Comment:
Let's be consistent with the name of the volume as later on we have: ```kubectl cloudflow deploy  ... file-ingress.source-data-mount=claim1 index.docker.io/<user>/...```. Either use `claim1` or change the commands bellow.

Reply:
Fixed